h/views/badge.py

```diff
@@ -30,7 +31,10 @@ def badge(request):
     those pages. The Chrome extension is oblivious to this, we just tell it
     that there are 0 annotations.
 
+    NewRelic APM has been disabled for this function.
+
     """
+    newrelic.agent.ignore_transaction(flag=True)
```

Comment:
```suggestion
    """
    # Disable NewRelic APM for this function.
    newrelic.agent.ignore_transaction(flag=True)
```

This is good to know, but not all that relevant to the caller of the function, so probably a comment rather than docstring

Reply:
Thanks Jon - Code has been updated.